azerothcore/azerothcore-wotlk
# fix(Scripts/Player): some refactor related to PlayerCommand functions

This PR is meant to refactor PlayerCommand function calls. It makes no impact of current .learn & .unlearn commands behavior

## Changes Proposed:
-  Replaced PlayerCommand class with Acore::PlayerCommand namespace
-  Renamed PlayerCommand functions to be more readable
- Removed non needed PlayerCommand instances
- Relisted headers to correctly optimize compilation

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- tested build
- tested .learn & .unlearn commands


## How to Test the Changes:


1. Choose any wanted spell (please don't try server side spells... :D)
2. Trigger .learn <spellID> and observe details. Spell should be added in spell book
3. Trigger .unlearn <spellID> and observe details. Spell should be removed from spell book

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
